libs/build_api_v2.py

```diff
@@ -51,11 +61,13 @@ def _build_actuals(actual_data: dict) -> Actuals:
 
 
 def build_region_summary(
-    latest_values: dict,
+    one_region: timeseries.OneRegionTimeseriesDataset,
     latest_metrics: Optional[Metrics],
     risk_levels: RiskLevels,
-    region: pipeline.Region,
+    log,
```

Comment:
I know I bring this up every once in a while, but do we gain much from passing log through here? I find that passing log through bleeds the intentions of the calling function into sub functions - I don't think that it should be a requirement that if I want to build a region summary I need to have my own logging context set up.